sync.js

```diff
@@ -100,6 +102,9 @@ class SyncState {
   }
 
   stale (peer) {
+    if (peer.state.topic === ReplicationState.STARTED) return true
+    if (peer.state.topic === ReplicationState.ERROR) return false
```

Comment:
should this be false??

Reply:
A peer is only stale when it's in progress but has stopped sending progress updates, right? Why would STARTED or ERROR state peers be considered stale?